### PR TITLE
'Py-create-file-from-terminal Solution'

### PR DESCRIPTION
**Pull request by MilManIT**

### CODE REVIEW

File: app/create_file.py [hunk 1]
```diff
@@ -1 +1,33 @@
...
+            break
+        lines.append(line)
+
+    timestamp = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+    lines = [f"{i} {line}" for i, line in enumerate(lines, start=1)]
+    content = f"{timestamp}\n" + "\n".join(lines) + "\n"
+
+    mode = "a" if os.path.isfile(file_path) else "w"
+    with open(file_path, mode) as f:
+        f.write(content)
```
Comment (ArsenPidhoretskyi) [inline app/create_file.py:33]: Make it as function

File: app/create_file.py [hunk 2]
```diff
@@ -1 +1,33 @@
...
+import sys
+import os
+from datetime import datetime
+
+args = sys.argv[1:]
+dir_path = ""
+file_name = ""
+file_content = ""
+
+if "dir" in args:
```
Comment (ArsenPidhoretskyi) [inline app/create_file.py:10]: Why you check by "dir"?
We have two flags: -f and -d

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]
Task is not properly done

File: app/create_file.py (hunk 2 above)
Comment (MilManIT) [reply app/create_file.py]: Because Olena Hul told me to do this

**Pull request closed without merging**